File: Web/RMA-Web-Shell/projects/medicare/src/app/medi-manager/models/icd10-code.ts
import { InjurySeverityTypeEnum } from "projects/claimcare/src/app/claim-manager/shared/enums/injury-severity-type-enum";
import { BodySideAffectedTypeEnum } from "projects/shared-models-lib/src/lib/enums/body-side-affected-type-enum";

export class ICD10Code {
     icd10CodeId: number;
     icd10Code: string;
     icd10CodeDescription: string;
     icd10SubCategoryId: number;
     icd10SubCategoryDescription: string;
     icd10DiagnosticGroupId: number;
     icd10CategoryId: number;
     bodySideAffected: BodySideAffectedTypeEnum;
     severity: InjurySeverityTypeEnum;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/ICD10Code.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class ICD10Code : AuditDetails
    {
        public int Icd10CodeId { get; set; }
        public string Icd10Code { get; set; }
        public string Icd10CodeDescription { get; set; }
        public int Icd10SubCategoryId { get; set; }
        public string Icd10SubCategoryDescription { get; set; }
        public int Icd10DiagnosticGroupId { get; set; }
        public int Icd10CategoryId { get; set; }
        public BodySideAffectedTypeEnum? BodySideAffected { get; set; }
        public InjurySeverityTypeEnum? Severity { get; set; }
    }
}
